frontend/src/views/ShootList.vue

```diff
@@ -460,8 +543,18 @@ export default {
       })
     },
     allHeaders () {
-      const allHeaders = [...this.standardHeaders, ...this.customHeaders]
-      return sortBy(allHeaders, ['weight', 'text'])
+      let allHeaders = [...this.standardHeaders, ...this.customHeaders]
+      allHeaders = sortBy(allHeaders, ['weight', 'text'])
+      if (this.freezeSorting) {
+        return map(allHeaders, header => {
```

Comment:
I think this magic is done because otherwise we loose reactivity. We are mixing header configuration with reactive table header properties. I have the impression that in two weeks nobody will understand why do we create the deep copy here.